src/main/java/by/custom/utilcalculator/controller/MessageRouter.java

```diff
@@ -35,36 +35,47 @@ public SendMessage route(final Update update) {
 
     //we receive message from user, check it and handle it if it is ok here
     private SendMessage getCheckInputMessageAndGetAnswer(final Message message) {
-        String usersMessage = message.getText();
-        String chatID = message.getChatId().toString();
-        String answer;
+        final String usersMessage = message.getText();
+        final String chatID = message.getChatId().toString();
+        final String answer;
         try {
             answer = route(usersMessage, chatID);
-        } catch (UtilsborException e) {
+        } catch (final UtilsborException e) {
             e.printStackTrace();
             return new SendMessage(chatID, getExceptionText(e));
         }
         return new SendMessage(chatID, answer);
     }
 
     private String route(final String usersMessage, final String chatID) throws UtilsborException {
-        String answer;
-        switch (usersMessage) {
-            case Command.START -> {
+        final String answer;
+        Command requestingCommand = Command.getCommandByKey(usersMessage);
+        switch (requestingCommand) {
+            case START -> {
                 userProgressManager.createNewUserProgress(chatID);
                 answer = getGreetingMessage();
             }
-            case Command.EAES, Command.OTHER_COUNTRIES ->
-                    answer = userProgressManager.processCarOrigin(usersMessage, chatID);
-            case Command.PHYSICAL_PERSON, Command.JURIDICAL_PERSON ->
-                    answer = userProgressManager.processOwnerType(usersMessage, chatID);
-            case Command.LESS_3_YEARS_AGE, Command.BETWEEN_3_AND_7_YEARS_AGE ->
-                    answer = userProgressManager.processCarAge(usersMessage, chatID);
-            case Command.GASOLINE_TYPE_ENGINE, Command.ELECTRIC_TYPE_ENGINE ->
-                    answer = userProgressManager.processEngineType(usersMessage, chatID);
-            case Command.VOLUME_LESS_1000_CM, Command.VOLUME_BETWEEN_1000_2000_CM, Command.VOLUME_BETWEEN_2000_3000_CM,
-                    Command.VOLUME_BETWEEN_3000_3500_CM, Command.VOLUME_MORE_3500_CM ->
-                    answer = userProgressManager.processEngineVolume(usersMessage, chatID);
+
+            case EAES, OTHER_COUNTRIES -> answer = userProgressManager.processCarOrigin(requestingCommand, chatID);
```

Comment:
What about `break`? Is it not needed?

Reply:
Thanks for asking! I have googled it and discovered this.

1. If we are using traditional switch-case block like 
switch (smth) {
case 1: 
           dosmth(1;) 
           break;
case 2: 
           dosmth(2); 
           break;
case 3:
   dosmth(3);
   break;
}
Using break; is NECESSARY to avoid doing useless code.

2. If we use arrows like 
switch(smth) {
case 1 -> dosmth(1);
case 2 -> dosmth(2);
case 1 -> dosmth(3);
}
Using break; is NOT necessary. 

I didn't know about it.